core/src/test/java/org/apache/druid/math/expr/FunctionTest.java

```diff
@@ -407,12 +407,40 @@ public void testRoundWithExtremeNumbers()
     assertExpr("round(CAST(minLong, 'DOUBLE') - 1, -2)", BigDecimal.valueOf(((double) Long.MIN_VALUE) - 1).setScale(-2, RoundingMode.HALF_UP).doubleValue());
   }
 
+  @Test
+  public void testRoundWithNullValue()
+  {
+    Set<Pair<String, String>> invalidArguments = ImmutableSet.of(
+        Pair.of("null", "STRING"),
+        Pair.of("x", "STRING")
+    );
+    for (Pair<String, String> argAndType : invalidArguments) {
+      if (NullHandling.sqlCompatible()) {
+        assertExpr(String.format(Locale.ENGLISH, "round(%s)", argAndType.lhs), null);
```

Comment:
nit: could use `StringUtils.format(...)`

Reply:
Thanks for your tips .Actually, I think it can be more flexable to handle null value. If we  handle it like:

      if (NullHandling.sqlCompatible() && value1.isNumericNull()) {
        return ExprEval.of(null);
      }
      if (value1.type() != ExprType.LONG && value1.type() != ExprType.DOUBLE) {
        throw new IAE(
            "The first argument to the function[%s] should be integer or double type but got the type: %s",
            name(),
            value1.type()
        );
      }

It is not compatible for usage  `round(null,  1)` , which  should assign a default  value for null.  I am trying to be compatible for usages like `round(null)` and `round(null, 1)`

Reply:
Sorry, I don't understand what you mean. Could you explain more details?

Reply:
Oh, I meant we have a utility function that does this format with locale stuff, https://github.com/apache/druid/blob/master/core/src/main/java/org/apache/druid/java/util/common/StringUtils.java#L166 that could be used to make this a bit shorter, but since is a test it isn't a big deal whether or not you change to use it

Reply:
ok, I have changed it.